sdk/trace/id_generator_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	// Can't assert values of IDs but can assert they're valid, try a lot as a sort of fuzz check.
+	_tryTimes    = 1000
```

Comment:
Can we move it to the `TestNewIDs` and rename it to `n` (to make it similar to `b.N` when doing benchmarks)?

PS. In Go we do not start the names with underscore (`_`)

Reply:
Thanks @pellared.
I moved it to `TestNewIDs` and `TestNewSpanID` and rename it to `n`.

But I didn't quite get:
> to make it similar to b.N when doing benchmarks

Do you have an example?

Reply:
> Do you have an example?

https://pkg.go.dev/testing#B